packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2610,7 +2610,7 @@ export default mixins(
 					});
 				});
 			},
-			removeNode(nodeName: string, trackHistory = false) {
+			removeNode(nodeName: string, trackHistory = false, partOfBulkDelete = false) {
```

Comment:
would rename this prop to something more apt to this context and clear `trackBulk = true`

Reply:
Ok.